### PR TITLE
[Web] Update equilibrium phase benchmark

### PR DESCRIPTION
**Pull request by renchao-lu**
As titled. 

- The up-to-date database file offered by PSI is now applied for the equilibrium phase benchmark. This update raises a slight difference to the previous simulation results and thus the expected result files are updated as well. 
- Benchmark description is enriched.

### CODE REVIEW

File: web/content/docs/benchmarks/reactive-transport/calcite.pandoc [hunk 1]
```diff
@@ -16,22 +16,61 @@ title = "Advective flow and precipitation/dissolution equilibrium reactions"
...
 
 ## Model setup
 
-The operator-splitting (OS) model formulation (Strang, 1968) with a sequential non-iterative approach (SNIA) has been used to implement the reactive component transport processes in saturated porous media illustrated in this benchmark. In a ﬁrst step, the mass balance equations describing each component transport present in the fluid is solved according to
+A one-dimensional (1D) model domain of 0.5 m discretized into 100 uniform elements has been selected for the spatial discretization of the system. Dirichlet (constant concentration) and Neumannn (no flux) boundary condition are defined for the upstream inflow and the downstream, respectively. A longitudinal dispersivity of 0.0067 m. A time step size of 100 s
```
Comment (endJunction) [inline web/content/docs/benchmarks/reactive-transport/calcite.pandoc]:
```suggestion
An one-dimensional (1D) model domain of 0.5 m discretized into 100 uniform elements has been selected for the spatial discretization of the system. Dirichlet (constant concentration) and Neumannn (no flux) boundary condition are defined for the upstream inflow and the downstream, respectively. A longitudinal dispersivity of 0.0067 m. A time step size of 100 s
```

File: web/content/docs/benchmarks/reactive-transport/calcite.pandoc [hunk 2]
```diff
@@ -16,22 +16,61 @@ title = "Advective flow and precipitation/dissolution equilibrium reactions"
...
 
+Thermodynamic data for hydrolysis, aqueous speciation and dissolution/precipitation reactions between Mg, Ca, Cl and carbonate were selected from version 01/01 of the NAGRA/PSI chemical thermodynamic database (Hummel et al. 2002). Although, several other minerals containing Mg and Ca were available in the NAGRA/PSI database (i.e. magnesite), only two solids were allowed to precipitate or dissolve in the studied system (calcite and dolomite (CaMg(CO$_3$)$_2$)).
```
Comment (endJunction) [inline web/content/docs/benchmarks/reactive-transport/calcite.pandoc]:
```suggestion
Thermodynamic data for hydrolysis, aqueous speciation, and dissolution/precipitation reactions between Mg, Ca, Cl and, carbonate were selected from version 01/01 of the NAGRA/PSI chemical thermodynamic database (Hummel *et al.* 2002). Although several other minerals containing Mg and Ca were available in the NAGRA/PSI database (*i.e.* magnesite), only two solids were allowed to precipitate or dissolve in the studied system (calcite and dolomite (CaMg(CO$_3$)$_2$)).
```

Comment (renchao-lu) [reply web/content/docs/benchmarks/reactive-transport/calcite.pandoc]: added.

**Comment by renchao-lu** [comment]
Please review the updated web content. @montoyav

**Comment by montoyav** [comment]
@renchao-lu done! Only minor things have been corrected.

File: web/content/docs/benchmarks/reactive-transport/calcite.pandoc (hunk 1 above)
Comment (endJunction) [reply web/content/docs/benchmarks/reactive-transport/calcite.pandoc]: a one

**Pull request merged**